src/webcomponents/helperfunctions/jsonToCol.ts

```diff
@@ -0,0 +1,34 @@
+import { JsonToCol, LanguagesInput } from "../../@types/interfaces";
+
+const mapLangToSheetColumns = (languages: LanguagesInput) => {
+  const columnTitle = Object.keys(languages).map((title) => ({
+    type: "text",
+    title,
+    width: 150,
+  }));
+  return columnTitle;
+};
+
+const mapLangToSheetData = (languages: LanguagesInput) => {
+  const messageIDs = Object.keys(languages).map((titles) => languages[titles]);
+  const firstLangMessageIDs = Object.keys(messageIDs[0]);
+  const rows = messageIDs.map((messageID: object) =>
+    Object.entries(messageID).map((row) => {
+      return `${row[0]}:${row[1]}`;
```

Comment:
it is very strange to serialize this string and it is un necessary

Reply:
I did this to validate the inputs(if you put budget is position 0 on 1st array and the put budget in position 1 in 2nd array it will not work)